.envrc

```diff
@@ -1,28 +1,25 @@
 # PostgreSQL
-export TEST_POSTGRES_BASE_URI="postgres://prisma:prisma@localhost:5432"
 export TEST_POSTGRES_URI="postgres://prisma:prisma@localhost:5432/tests"
-# Note: the isolated instance is only needed for one test (client/src/__tests__/integration/errors/connection-limit-postgres/test.ts)
-export TEST_POSTGRES_ISOLATED_URI="postgres://prisma:prisma@localhost:5435/tests"
 export TEST_POSTGRES_URI_MIGRATE="postgres://prisma:prisma@localhost:5432/tests-migrate"
 export TEST_POSTGRES_SHADOWDB_URI_MIGRATE="postgres://prisma:prisma@localhost:5432/tests-migrate-shadowdb"
+# Note: the isolated instance is only needed for one test (client/src/__tests__/integration/errors/connection-limit-postgres/test.ts)
+export TEST_POSTGRES_ISOLATED_URI="postgres://prisma:prisma@localhost:5435/tests"
 
 # MySQL
-export TEST_MYSQL_BASE_URI="mysql://root:root@localhost:3306"
 export TEST_MYSQL_URI="mysql://root:root@localhost:3306/tests"
-# Note: the isolated instance is only needed for one test (client/src/__tests__/integration/errors/connection-limit-mysql/test.ts)
-export TEST_MYSQL_ISOLATED_URI="mysql://root:root@localhost:3307/tests"
 export TEST_MYSQL_URI_MIGRATE="mysql://root:root@localhost:3306/tests-migrate"
 export TEST_MYSQL_SHADOWDB_URI_MIGRATE="mysql://root:root@localhost:3306/tests-migrate-shadowdb"
+# Note: the isolated instance is only needed for one test (client/src/__tests__/integration/errors/connection-limit-mysql/test.ts)
+export TEST_MYSQL_ISOLATED_URI="mysql://root:root@localhost:3307/tests"
 
 # MariaDB
-export TEST_MARIADB_BASE_URI="mysql://root:root@localhost:4306"
-export TEST_MARIADB_URI="mysql://prisma:prisma@localhost:4306/tests"
+export TEST_MARIADB_URI="mysql://root:root@localhost:4306/tests"
```

Comment:
```suggestion
# Note that the database name must be `tests` (in our test setup we search and replace that value later)
export TEST_MARIADB_URI="mysql://root:root@localhost:4306/tests"
```

Reply:
Decided to add a note at top of file instead as otherwise we need to repeat this one too many times all over the files